WcaOnRails/app/assets/stylesheets/variables.scss

```diff
@@ -1,4 +1,5 @@
 // ***** Overriding Bootstrap variables ****
 $link-color: #f60;
 
-$past-member: #fecd55;
\ No newline at end of file
+$past-member: #fecd55;
```

Comment:
Let's call all color variables `$*-color`, so in this case, `$past-member-color`